templates/include/telemetry/mainPayload.1.schema.json

```diff
@@ -109,7 +120,51 @@
     },
     "lateWrites": {
       "type": "object",
-      "properties": { }
+      "description": "This sections reports writes to the file system that happen during shutdown. The reported data contains the stack and the file names of the loaded libraries at the time the writes happened.",
+      "properties": {
+        "memoryMap": {
+          "type": "array",
+          "description": "entries in the format [module name, breakpad identifier]",
+          "items": {
+            "type": "array",
+            "items": [
```

Comment:
This representation will cause these to be anonymous structs in BQ. If we set it simply to `"items": {"type": "string"}`, it will be a string array. Does that seem reasonable since the types are all the same?

Reply:
For this and `stacks` below.

Reply:
Yeah, that's cleaner. Good suggestion!